devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -84,23 +84,41 @@ metadata {
 	}
 }
 
+// Perform a periodic check to ensure that initialization of the device was successful
+def getInitVerifyCheckPeriodicSecs() {
```

Comment:
We typically name these functions in all caps to make them look like constants, for example `getINIT_VERIFY_CHECK_PERIOD_SECS`

Reply:
Aaah, good point. Missed it.